src/components/main/main.js

```diff
@@ -1,7 +1,15 @@
 import { Component } from "react";
-import { Container, Row, Col, Button, Modal, Form, Table } from "react-bootstrap";
```

Comment:
@rainbow-catch plz keep the original code for this line,
The original one looks better